lib/zinnia.js

```diff
@@ -294,12 +295,6 @@ export async function run ({
       resetTimeout()
       process.stderr.write(data)
     })
-
-    childProcess.on('exit', (code, signal) => {
-      const exitReason = signal ? `via signal ${signal}` : `with code: ${code}`
-      const msg = `${capitalize(module)} exited ${exitReason}`
```

Comment:
We are losing the information on whether the module exited via a signal or by returning a (potentially non-zero) exit code. Can we preserve it, please?

Reply:
Oh, I see. This code part of our codebase is truly difficult to understand 😢

Reply:
Yeah, that's I think why this was unnoticed for a while

Reply:
This is taking a step in the right direction, by removing some branches :)